tests/MabeEnumTest/EnumTest.php

```diff
@@ -385,6 +385,19 @@ public function testNotUnserializable()
         unserialize('O:' . strlen(EnumBasic::class) . ':"' . EnumBasic::class . '":0:{}');
     }
 
+    public function testHasName()
+    {
+        $enum = EnumBasic::ONE();
+
+        $this->assertFalse($enum->hasName(''));
+        $this->assertFalse($enum->hasName(false));
+        $this->assertFalse($enum->hasName(true));
+        $this->assertFalse($enum->hasName('str'));
+        $this->assertFalse($enum->hasName(new \stdClass()));
+        $this->assertTrue($enum->hasName('ONE'));
+        $this->assertTrue($enum->hasName('STR'));
```

Comment:
That's a static method -> you don't need to instantiate a new enumeration -> just use `EnumBasic::hasName()`